lib/api/controllers/adminController.js

```diff
@@ -119,11 +120,34 @@ function AdminController(kuzzle) {
         modifiedRequestObject = newRequestObject;
 
         _.forEach(kuzzle.pluginsManager.plugins, (object, id) => {
-          pluginsConfig[id] = _.assignIn({}, object);
-          if(pluginsConfig[id].object.context && pluginsConfig[id].object.context.config) {
+          pluginsConfig[id] = JSON.parse(stringifySafe(object));
```

Comment:
I am not very conmfortable with dumping some objects we cannot have control on.
The debug use case is already handled by the core dumps and getting the plugins configuration from the db is already a valuable information to me.